todos/src/components/Todos.js

```diff
@@ -5,24 +5,31 @@ import "styles/todos.css";
 import { useSelector } from "react-redux";
 import EmptyTaskList from "components/EmptyTaskList";
 import LoadMoreBtn from "components/LoadMoreBtn";
-import { PER_PAGE } from "constants";
+import { LOAD_MORE, SHOW_LESS, PER_PAGE } from "constants";
 
 function Todos() {
   const list = useSelector((state) => state.todoReducers.list);
+  const filter = useSelector((state) => state.filterReducer.filter);
   const isAddTaskVisible = useSelector(
     (state) => state.toggleReducers.isAddTaskVisible
   );
   const currentPage = useSelector(
     (state) => state.currentPageReducer.currentPage
   );
-  const displayedTodoList = list.slice(0, PER_PAGE * currentPage);
+  const filteredTodos =
+    filter === "all"
+      ? list
+      : filter === "incomplete"
+      ? list.filter((todo) => !todo.isCompleted)
+      : list.filter((todo) => todo.isCompleted);
```

Comment:
This is hard to read. Let's do it this way
```suggestion
    if( filter === ALL ){
      return list
    }
    
    if( filter === INCOMPLETE ) {
      return list.filter((todo) => !todo.isCompleted)
    }

    return list.filter((todo) => todo.isCompleted)
```

Reply:
Alright, @hhu-wtag Bhai. Done.